File: GestureDetector/app/src/main/java/com/euzl/gesturedetector/MainActivity.java
package com.euzl.gesturedetector;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button toastButton = findViewById(R.id.toast_button);
        Button textButton = findViewById(R.id.textview_button);

        toastButton.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, ToastActivity.class)));
    }
}

File: EditText/app/src/main/java/com/euzl/edittext/MainActivity.kt
package com.euzl.edittext

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.euzl.edittext.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.buttonSetText.setOnClickListener { startActivity(Intent(this, SetTextActivity::class.java)) }
        binding.buttonNotModify.setOnClickListener { startActivity(Intent(this, NotModifyTextActivity::class.java)) }
        binding.buttonTextChangeEvent.setOnClickListener { startActivity(Intent(this, TextChangeEventActivity::class.java)) }
    }
}